backend/database/factories/UserFactory.php

```diff
@@ -18,11 +18,11 @@ class UserFactory extends Factory
     public function definition()
     {
         return [
-            'name' => fake()->name(),
+            'name' => fake()->unique()->name(),
             'email' => fake()->safeEmail(),
+            'avatar_url' => fake()->unique()->imageUrl(640, 480),
```

Comment:
is this being referenced in the UserItem component? diba there is an error encountered? can you use what we discussed this morning where this will be static and just upload the image to your files?